backend/README.md

```diff
@@ -53,6 +53,27 @@ can issue GraphQL requests or access GraphQL Playground in the browser.
 
 ![GraphQL Playground](../.gitbook/assets/graphql-playground.png)
 
+### Database Indices and Constraints
+
+Database indices and constraints need to be created when the database and the
+backend is running:
+
+{% tabs %}
+{% tab title="Docker" %}
+```bash
+docker-compose exec backend yarn run db:migrate init
+```
+{% endtab %}
+
+{% tab title="Without Docker" %}
+```bash
+# in folder backend/
+# make sure your database is running on http://localhost:7474/browser/
+yarn run db:migrate init
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-20T11:56:04Z" title="Monday, January 20th 2020, 12:56:04 pm +01:00">Jan 20, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -53,6 +53,27 @@ can issue GraphQL requests or access GraphQL Playground in the browser.
 
 ![GraphQL Playground](../.gitbook/assets/graphql-playground.png)
 
+### Database Indices and Constraints
+
+Database indices and constraints need to be created when the database and the
+backend is running:
+
+{% tabs %}
+{% tab title="Docker" %}
+```bash
+docker-compose exec backend yarn run db:migrate init
+```
+{% endtab %}
+
+{% tab title="Without Docker" %}
+```bash
+# in folder backend/
+# make sure your database is running on http://localhost:7474/browser/
+yarn run db:migrate init
```


when I ran this locally, without `Docker`... it seems to have run successfully, based on this output
```
yarn db:migrate init
yarn run v1.19.0
warning ../../package.json: No license field
$ yarn run __migrate --store ./src/db/migrate/store.js init
warning ../../package.json: No license field
$ migrate --compiler 'js:@babel/register' --migrations-dir ./src/db/migrations --store ./src/db/migrate/store.js init
  migrations dir : /home/mattwr18/code/Human-Connection/backend/src/db/migrations
Successfully created database indices and constraints!
  init : undefined
Done in 3.64s.
```